cmd/tdex/withdrawmarket.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"math"
+
+	pb "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/operator"
+	pbtypes "github.com/tdex-network/tdex-protobuf/generated/go/types"
+	"github.com/urfave/cli/v2"
+)
+
+var withdrawmarket = cli.Command{
+	Name:  "withdrawmarket",
+	Usage: "withdraw funds from some market.",
+	Flags: []cli.Flag{
+		&cli.Float64Flag{
+			Name:  "base_amount",
+			Usage: "the amount in BTC of base asset to withdraw from the market.",
+		},
+		&cli.Float64Flag{
+			Name:  "quote_amount",
+			Usage: "the amount in BTC of quote asset to withdraw from the market.",
+		},
+		&cli.StringFlag{
+			Name:  "address",
+			Usage: "the address where to send the withdrew amount(s).",
+		},
+		&cli.Int64Flag{
+			Name:  "millisatsperbyte",
+			Usage: "the mSat/byte to pay for the transaction",
+			Value: 100,
+		},
+	},
+	Action: withdrawMarketAction,
+}
+
+func withdrawMarketAction(ctx *cli.Context) error {
+	client, cleanup, err := getOperatorClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+
+	baseAsset, quoteAsset, err := getMarketFromState()
+	if err != nil {
+		return err
+	}
+	baseAmount := uint64(ctx.Float64("base_amount") * math.Pow10(8))
```

Comment:
not always true we have 8 decimals. maybe is better to use satoshis, still a dev tool